plugins/infinispan52/src/main/java/org/radargun/service/InfinispanServerLifecycle.java

```diff
@@ -11,47 +11,56 @@
 public class InfinispanServerLifecycle extends ProcessLifecycle<InfinispanServerService> {
    private boolean serverStarted;
    private boolean serverStopped;
-   private static final Pattern START_OK = Pattern.compile(".*\\[org\\.jboss\\.as\\].*started in.*");
-   private static final Pattern START_ERROR = Pattern.compile(".*\\[org\\.jboss\\.as\\].*started \\(with errors\\) in.*");
-   private static final Pattern STOPPED = Pattern.compile(".*\\[org\\.jboss\\.as\\].*stopped in.*");
 
    private volatile boolean gracefulStop = true;
 
    public InfinispanServerLifecycle(final InfinispanServerService service) {
       super(service);
    }
 
+   protected Pattern getStartOK() {
```

Comment:
Why isn't this using the `ServerStartPattern` enum?

Reply:
@alanfx I created `ServerStartPattern` on `plugin-infinispan100` containing only serverng pattern, if I add `plugin-infinispan100`on `plugin-infinispan52` I will introduce circular dependency between those modules